plugin/fzf.vim

```diff
@@ -367,7 +367,7 @@ try
   if !has_key(dict, 'source') && !empty($FZF_DEFAULT_COMMAND) && !s:is_win
     let temps.source = s:fzf_tempname()
     call writefile(s:wrap_cmds(split($FZF_DEFAULT_COMMAND, "\n")), temps.source)
-    let dict.source = (empty($SHELL) ? &shell : $SHELL).' '.fzf#shellescape(temps.source)
+    let dict.source = &shell.' '.fzf#shellescape(temps.source)
```

Comment:
`&shell` is set to `sh` so there's no user customization here. You could set/unset `$SHELL` instead.